churn-risk/src/app.py

```diff
@@ -1,184 +1,193 @@
-from test.e2e import walkthrough
-
 import pandas as pd
-from h2o_wave import app, main, Q, ui
-from plotly import graph_objects as go
+import os
 
+from pygments import highlight
+from pygments.formatters.html import HtmlFormatter
+from pygments.lexers import get_lexer_by_name
+from h2o_wave import app, main, Q, ui, data
 from .churn_predictor import ChurnPredictor
-from .config import Configuration
-from .plots import (
-    convert_plot_to_html,
-    generate_figure_pie_of_target_percent,
-    get_image_from_matplotlib,
-    tall_stat_card_dollars,
-    wide_stat_card_dollars,
-)
-from .utils import python_code_content
-
-config = Configuration()
+
+df = pd.read_csv('data/churnTest.csv')
+df.dropna(subset=['State', 'Account_Length', 'Area_Code', 'Phone_No'], inplace=True)
+df.fillna({'International_Plan': 'no', 'Voice_Mail_Plan': 'no', 'No_Vmail_Messages': 0, 'Total_Day_minutes': 0.00,
+           'Total_Day_Calls': 0, 'Day Charges': 0.00, 'Total_Eve_Minutes': 0.00, 'Total_Eve_Calls': 0,
+           'Evening Charges': 0, 'Total_Night_Minutes': 0.00, 'Total_Night_Calls': 0, 'Night Charges': 0.00,
+           'Total_Intl_Minutes': 0.00, 'Total_Intl_Calls': 0, 'Intl Charges': 0.00, 'No_CS_Calls': 0},
+          inplace=True)
+df['Total Charges'] = (df['Day Charges'] + df['Evening Charges'] + df['Night Charges'] + df['Intl Charges'])
+rank = df['Total Charges'].rank(pct=True).values[0]
 churn_predictor = ChurnPredictor()
 
 
-df = pd.read_csv(config.testing_data_url).head(40)
-df.fillna(config.def_column_values, inplace=True)
-df.dropna(subset=config.mandatory_columns, inplace=True)
-phone_choices = [ui.choice(name=str(phone), label=str(phone)) for phone in df[config.id_column]]
+def render_analysis(q: Q):
```

Comment:
This function does a lot, and I think it might be confusing to new people using this app to get started. WDYT of the render_analysis function calling a function for each piece of analysis: Shaply, PDP, and descriptive information?